docs/rules/comma-dangle.md

```diff
@@ -60,6 +60,17 @@ This rule has a string option or an object option:
 * `"always-multiline"` requires trailing commas when the last element or property is in a *different* line than the closing `]` or `}` and disallows trailing commas when the last element or property is on the *same* line as the closing `]` or `}`
 * `"only-multiline"` allows (but does not require) trailing commas when the last element or property is in a *different* line than the closing `]` or `}` and disallows trailing commas when the last element or property is on the *same* line as the closing `]` or `}`
 
+Trailing commas when the last element is a spread are not allowed even when always is on.
```

Comment:
I think we should change the wording here a bit. The spec forbids trailing commas on AssignmentRestElement nodes, it's not that ESLint arbitrarily decided to not allow them for this rule in specific (see [this issue](https://github.com/eslint/eslint/issues/3627) for some more context). I would suggest something like "Please note that trailing commas are not allowed by the spec".